src/devices/machine/cat702.h

```diff
@@ -23,13 +24,16 @@ class cat702_device : public device_t
 	void write_clock(int state);
 
 protected:
+	cat702_device(const machine_config &mconfig, device_type type, const char *tag, device_t *owner, uint32_t clock);
+
 	virtual void device_start() override;
 
-private:
 	uint8_t compute_sbox_coef(int sel, int bit);
 	void apply_bit_sbox(int sel);
 	void apply_sbox(const uint8_t *sbox);
 
+	static constexpr uint8_t initial_sbox[8] = { 0xff, 0xfe, 0xfc, 0xf8, 0xf0, 0xe0, 0xc0, 0x7f };
+
```

Comment:
Can you please put this in the .cpp file so isn’t unnecessarily exposed to downstream users?